bundles/framework/admin-announcements/view/AnnouncementsForm.jsx

```diff
@@ -0,0 +1,172 @@
+import React from "react";
+import "antd/dist/antd.css";
```

Comment:
We have wrapped individual AntD components under own components under scr/react/components that is exposed as "oskari-ui". One of the benefits is that you don't have to link the whole css just to get a style for a single component. You should use that approach here as well.

Reply:
apparently it was not needed anyways

Reply:
Maybe it's imported with other parts of the app using these components. Might be best to import the individual styles to be sure.

Reply:
done!